app/nexus/internal/state/backend/sqlite/persist/policy.go

```diff
@@ -156,26 +196,43 @@ func (s *DataStore) LoadPolicy(
 		return nil, fmt.Errorf("failed to load policy: %w", err)
 	}
 
-	// Deserialize permissions from comma-separated string
+	// Decrypt
+	decryptedSPIFFE, err := s.decrypt(encryptedSPIFFE, nonce)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decrypt SPIFFE pattern: %w", err)
+	}
+	decryptedPath, err := s.decrypt(encryptedPath, nonce)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decrypt Path pattern: %w", err)
+	}
+
+	decryptedPermissions, err := s.decrypt(encryptedPermissions, nonce)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decrypt permissions: %w", err)
+	}
+
+	// Set decrypted values
+	policy.SPIFFEIDPattern = string(decryptedSPIFFE)
+	policy.PathPattern = string(decryptedPath)
```

Comment:
nit: rename as `decryptedSPIFFEIDPattern` `decryptedPathPattern` etc -- pattern "implies" that this is a regex pattern; having clearer variable names will make the intention clear too.

--

For e.g. in context:

path: `tenants/db/credentials`

pathPattern: `^tenants/db/.*$`